<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>participarLeccion</class>
 <widget class="QDialog" name="participarLeccion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coline - 201513696</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Participar en la Leccion</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Titulo</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="txtTituo"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Explicacion</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="txtExplicacion"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Enunciado</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTextEdit" name="txtEnunciado"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnCalificar">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>480</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEvaluar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>21</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>511</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Solucion</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTextEdit" name="txtSolucion"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>380</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Ejemplo</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QTextEdit" name="txtEjemplo"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
